activetable.html

```diff
@@ -4,12 +4,18 @@
     <meta http-equiv="content-type" content="text/html; charset=UTF-8">
     <meta charset="UTF-8">
     <link href="activetable.css" rel="stylesheet" type="text/css">
+    <style id="row-height-style"></style>
     <script src="activetable.js"></script>
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js"></script>
```

Comment:
I thought using the same jquery version as edx-platform itself is saner for something running within edx-platform, but I don't have a real technical reason for this change.  I'm happy to revert it if you prefer.